src/main/java/games/strategy/triplea/ui/display/TripleADisplay.java

```diff
@@ -25,10 +25,7 @@ public TripleADisplay(final TripleAFrame ui) {
   }
 
   @Override
-  public void initialize(final IDisplayBridge bridge) {
-    final IDisplayBridge m_displayBridge = bridge;
-    m_displayBridge.toString();
```

Comment:
The only implementation of `IDisplayBridge` is `DefaultDisplayBridge`, and it does not override `Object#toString()`.  Therefore, it doesn't seem that any side effects could be caused by calling this method, so it should be safe to remove, and thus remove the `m_displayBridge` local variable in the process.